Makefile.common

```diff
@@ -109,7 +109,7 @@ endif
 %: common-% ;
 
 .PHONY: common-all
-common-all: precheck style check_license lint unused build test
+common-all: precheck style check_license lint unused build test react-app-test
```

Comment:
Isn't this file propagated to other repos that would not have `react-app-test`?

Reply:
Whoops of course - I carefully avoided modifications to it up until the last changes where I stopped thinking :D Will look & fix tomorrow.